.gitignore

```diff
@@ -28,4 +29,8 @@ coverage
 *.sln
 *.sw?
 
+# capacitor
 /capacitor.config.ts
+/android
```

Comment:
Do not need to ignore `android` and `ios` directories as long as custom native code will be inside those directories.